setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
I actually want to test if I can make `base.txt` do `-e setup.py` since that's more explicit than `-e .`